app/models/like.rb

```diff
@@ -1,4 +1,5 @@
 class Like < ApplicationRecord
   belongs_to :answer
   belongs_to :user
+
```

Comment:
Mesmo que o Rails coloque o belongs_to como obrigatório, recomendo colocar a validação de presença no `answer_id` e no `user_id`. Aí se tu fizer um `Like.create(answer: Answer.new, user: User.new)`, o rails vai tratar o erro, ao invés de lançar uma exceção de not null do banco de dados.

Reply:
Recomendo fazer isso para os demais belongs_to também.